CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ scheme are considered to be bugs.
 
 ### Changed
 
-* Your contribution here.
+* [#521](https://github.com/hashie/hashie/pull/499): TODO: Add desc of change and make major version bump! - [@carolineartz](https://github.com/carolineartz)
```

Comment:
Missing a period, so the bot complains. Leave Your contribution here alone, too.